src/twisted/words/test/test_irc.py

```diff
@@ -1602,9 +1615,9 @@ def _createHeartbeat():
         self.assertEqual(self.client.hostname, 'foo')
 
         # Pump the clock enough to trigger one LoopingCall.
-        self.assertEqual(self.transport.value(), '')
+        assertEqualBufferValue(self.transport.value(), '')
```

Comment:
this way looks a little out of place. I think it would be better to have a helper function to only convert to utf-8 unicode string and use `assertEqual`  with the utf-8 string